pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/alderspensjon/HjemlerAP.kt

```diff
@@ -0,0 +1,277 @@
+package no.nav.pensjon.brev.maler.fraser.alderspensjon
+
+import no.nav.pensjon.brev.api.model.AlderspensjonRegelverkType
+import no.nav.pensjon.brev.api.model.AlderspensjonRegelverkType.AP2016
+import no.nav.pensjon.brev.api.model.AlderspensjonRegelverkType.AP2025
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.Language.Nynorsk
+import no.nav.pensjon.brev.template.OutlinePhrase
+import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
+import no.nav.pensjon.brev.template.dsl.expression.and
+import no.nav.pensjon.brev.template.dsl.expression.expr
+import no.nav.pensjon.brev.template.dsl.expression.ifNull
+import no.nav.pensjon.brev.template.dsl.expression.isNotAnyOf
+import no.nav.pensjon.brev.template.dsl.expression.isOneOf
+import no.nav.pensjon.brev.template.dsl.expression.not
+import no.nav.pensjon.brev.template.dsl.expression.or
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+
+
+data class HjemlerInnvilgelseForAP2011AP2016(
+    val garantipensjonInnvilget: Expression<Boolean>,
+    val godkjentYrkesskade: Expression<Boolean>,
+    val innvilgetFor67: Expression<Boolean>,
+    val pensjonstilleggInnvilget: Expression<Boolean>,
+    val regelverkType: Expression<AlderspensjonRegelverkType>,
+) : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        showIf(regelverkType.isNotAnyOf(AP2025)) {
```

Comment:
Er det ei problemstilling her med AP1967, eller er ikkje denne sekvensen relevant for 1967-kontekst?

Reply:
AP1967 skal egentlig ikke være relevant lenger, men jeg utvider logikken mend betingelsen not(AP1967). Bra spottet!